### DISCUSSION TITLE
Emit to parent

### DISCUSSION THREAD

**Discussion by pascallieverse** [livewire/livewire#4215] [3. Feature ideas]
Currently its possible to scope the event to parent listeners with emitUp. But when you have multiple listeners with the same name its emitting to all components above it.

I would love to see a functionality emitToParent() where its sending the event only to the parent above and not all the other components above the parent.

**Comment by joshhanley** [comment]
@pascallieverse yeah that could be helpful!

For a work around, have a look at this PR #3916, the PR is a little different than what you are after, but in one of the comments I describe how to scope listeners.

Hope this helps!
